### ISSUE TITLE
Multiple query parameters: only last one included in type

### ISSUE THREAD

**Issue by jessecoleman** [Bewinxed/svetch#6]
When generating the types for a route that accepts multiple query parameters, it seems like it only includes the last declared one in the API type:

```
export const GET: RequestHandler = async ({ url }) => {
    const mode = url.searchParams.get('mode');
    const date = url.searchParams.get('date');
}
```

![image](https://github.com/user-attachments/assets/c99c0996-02f0-471c-9c59-d08b80eb34f6)

swapping order:

```
export const GET: RequestHandler = async ({ url }) => {
    const date = url.searchParams.get('date');
    const mode = url.searchParams.get('mode');
}
```

![image](https://github.com/user-attachments/assets/30d02699-f1d8-432a-bff8-af7ed8e651ef)

**Comment by Bewinxed** [comment]
Hello, It seems to work here, can you double check if you're using the
latest version? ***@***.***

if you use Bunx sometimes it doesn't install the latest version.

try with npx ***@***.*** and let me know.

which node version & OS?

**Comment by jessecoleman** [comment]
I am using the latest version:
```
npx svetch.ts@latest
Need to install the following packages:
svetch.ts@2.1.45

node --version
v20.11.1

OS: Windows subsystem for linux
```

Still seeing the issue locally, on multiple routes. 

Another example (abridged):
```
export const GET: RequestHandler = async ({ url, locals }) => {
  if (!locals.user) {
    error(403, 'not logged in');
  }
  const userId = locals.user.id;
  const stat = url.searchParams.get('stat');
  const mode = url.searchParams.get('mode');
  const puzzle_id = url.searchParams.get('puzzleId');
}
```
 
![image](https://github.com/user-attachments/assets/e4f55d45-75ca-4beb-a343-30e98b8412b1)

**Comment by Bewinxed** [comment]
Found the issue, fixed in 2.1.46

**Comment by jessecoleman** [comment]
Wow! Thanks for the quick fix!
